server/index.js

```diff
@@ -71,15 +71,40 @@ io.on('connection', (socket) => {
     });
 
      // received signal to start a session from the host of a room, emit redirect signal to all guests and host
-     socket.on('I am emitted from an imported socket!', () => {
-        console.log('successfully received emit from imported socket in swiping page')
+     socket.on('initialize_room', ({roomId, numGuests, movies}) => {
+        console.log('INITIALIZE ROOM:');
+        console.log(roomId);
+        console.log(numGuests);
```

Comment:
I think we should take out these console.logs, or maybe combine them into one comma separated console.log() with all the variables we care about? What do you think?

Reply:
Sounds good. Will just use template literals to have these variables shown with console log statements we have!